pkg/backup/item_backupper.go

```diff
@@ -97,19 +104,19 @@ type ItemBackupper interface {
 }
 
 type defaultItemBackupper struct {
-	backup          *api.Backup
-	namespaces      *collections.IncludesExcludes
-	resources       *collections.IncludesExcludes
-	backedUpItems   map[itemKey]struct{}
-	actions         []resolvedAction
-	tarWriter       tarWriter
-	resourceHooks   []resourceHook
-	dynamicFactory  client.DynamicFactory
-	discoveryHelper discovery.Helper
-	snapshotService cloudprovider.SnapshotService
-
+	backup                  *api.Backup
+	namespaces              *collections.IncludesExcludes
+	resources               *collections.IncludesExcludes
+	backedUpItems           map[itemKey]struct{}
+	actions                 []resolvedAction
+	tarWriter               tarWriter
+	resourceHooks           []resourceHook
+	dynamicFactory          client.DynamicFactory
+	discoveryHelper         discovery.Helper
+	snapshotService         cloudprovider.SnapshotService
 	itemHookHandler         itemHookHandler
```

Comment:
Do you mean fields directly passed into the constructor? bc in this example, `itemHookHandler` and `additionalItemBackupper` are both set in the constructor, just derived rather than directly passed in.